website/docs/security/tls.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 2
+description: Learn to secure connections using TLS
+---
+
+# Securing connections with TLS
```

Comment:
Should the file name here reflect the title of the docs, or that's okay?

http://localhost:3000/contributing/writing-guide/#names-and-urls

Reply:
They should match, but then I think the title should be shorter

Reply:
Okay. Shorter it is! We can update the documentation to cover instances when the titles are long.

Reply:
What title should we use?

Reply:
That's file name. I want to make title shorter, not file name longer

Reply:
TLS Security or Securing with TLS

Reply:
Since TLS already is an acronym with security in it, TLS Connections may be better

Reply:
Oh, that's a good one!